test/onnx/test_onnx_opset.py

```diff
@@ -395,41 +396,6 @@ def forward(self, input):
         x = torch.randn(2, 3, 4)
         check_onnx_opsets_operator(MyModule(), x, ops, opset_versions=[9, 10])
 
-    def test_advanced_index(self):
```

Comment:
I felt this test is a little noisy. Also because the operator gets exported as more than 10 nodes it is hard to maintain. The coverage should be enough without this test given that there are more than 10 test cases for indexing in backend test.